src/main/java/lotto/controller/LottoApp.java

```diff
@@ -0,0 +1,54 @@
+package lotto.controller;
+
+import lotto.model.LottoNumbers;
+import lotto.model.LottoNumbersGenerator;
+import lotto.model.LottoNumbersGroup;
+import lotto.model.LottoResult;
+import lotto.model.Money;
+import lotto.model.Rate;
+import lotto.view.LottoAppInput;
+import lotto.view.LottoAppOutput;
+import lotto.view.dro.LottoResultDto;
+
+public class LottoApp {
+	private final LottoAppOutput lottoAppOutput;
+	private final LottoAppInput lottoAppInput;
+	private final LottoNumbersGenerator lottoNumbersGenerator;
+
+	public LottoApp(LottoAppOutput lottoAppOutput, LottoAppInput lottoAppInput,
+		LottoNumbersGenerator lottoNumbersGenerator) {
+		this.lottoAppOutput = lottoAppOutput;
+		this.lottoAppInput = lottoAppInput;
+		this.lottoNumbersGenerator = lottoNumbersGenerator;
+	}
+
+	public void run() {
+		Money inputMoney = inputMoney();
```

Comment:
```suggestion
		Money money = inputMoney();
```
입력 받은 money 객체라고 이름 지을 필요는 없지 않을까요?